tests/bson/bson-binary-set_state-001.phpt

```diff
@@ -25,27 +25,27 @@ foreach ($tests as $test) {
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-MongoDB\BSON\Binary::__set_state(array(
+%r\\?%rMongoDB\BSON\Binary::__set_state(array(
```

Comment:
I assume PHP 8.2+ now emits the FQCN with a leading backslash. Is that correct?

Reply:
Yes, that's correct. I opted for the `EXPECTF` regex over separate tests for this change, despite the rough look of the regular expression.